### ISSUE TITLE
Test hangs (intermitent)

### ISSUE THREAD

**Issue by sandreim** [paritytech/zombienet#105]
Seen this happening for https://github.com/paritytech/polkadot/pull/4859
Hanged CI job: https://gitlab.parity.io/parity/polkadot/-/jobs/1374388 

IIRC I have observed the hang locally but ruled it out at the time because I had my own set of changes. Not sure what's going on, but the parachains seem to be making progress and test is running - https://grafana.parity-mgmt.parity.io/explore?orgId=1&left=%5B%22now-3h%22,%22now%22,%22loki.parity-zombienet%22,%7B%22expr%22:%22%7Bpod%3D%5C%22zombie-6c8c8f76a0cc482932c2e418659f4fe4%2Falice%5C%22%7D%20%7C~%20%5C%22included%5C%22%22,%22refId%22:%22A%22,%22range%22:true%7D%5D

**Comment by pepoviola** [comment]
This should be fixed by https://github.com/paritytech/zombienet/pull/108, please let me know if popup again.
Thx!

**Comment by sandreim** [comment]
The issue is not fixed in 1.2.14, but now it never times out, it hangs forever. We need to check why this never times out or there are no side effects for the command/test from that happening - https://github.com/paritytech/zombienet/blob/main/src/networkNode.ts#L52

```
2022-02-23T17:54:47.234Z zombie::network-node Connecting api for alice at ws://10.24.1.241:9944...
2022-02-23 17:54:48 API/INIT: RPC methods not decorated: beefy_getFinalizedHead
2022-02-23T17:54:49.027Z zombie::network-node Connected to alice
✔ alice: parachain 2000 is registered within 60 seconds (3818ms)
2022-02-23T17:54:51.053Z zombie::network-node Connecting api for bob at ws://10.24.1.243:9944...
```

**Comment by sandreim** [comment]
https://github.com/paritytech/polkadot/pull/4797

**Comment by sandreim** [comment]
I think we should fix this in a way that we fail as early as possible. Now we still hang if for some reason a node doesn't start.

**Comment by pepoviola** [comment]
Yes, we should detect that the binary isn't running and exit asap.

Thanks!

**Comment by pepoviola** [comment]
Hi @sandreim, fixed in #131. Closing now.
Thx!
